src/Router/ModuleFileRouter.php

```diff
@@ -81,12 +83,17 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
         );
 
         if (is_file($fullpath)) {
-            return (new \Zend\Diactoros\Response)
+            $resp = (new \Zend\Diactoros\Response)
                 ->withStatus(200)
                 ->withBody(new \Zend\Diactoros\Stream($fullpath));
+            if ($this->contenttype != "") {
+                $resp = $resp->withHeader("Content-Type", $this->contenttype);
+            }
+            return $resp;
         }
         return (new \Zend\Diactoros\Response())
             ->withStatus(404)
-            ->withBody(new StringStream($fullpath . ": File not found"));
+            ->withBody(new StringStream($fullpath . ": File not found"))
```

Comment:
To use the template, should be :
`return (new \LORIS\Http\Error($request, 404, $fullpath . ": File not found"));`